tox.ini

```diff
@@ -10,6 +10,8 @@ depends =
 skip_install = true
 ignore_errors = true
 commands =
+    docker build --build-arg PYTHON_VERSION=3.8 -t taskhuddler-{envname}-py38-test -f Dockerfile.test .
```

Comment:
Thanks for tagging along the 3.8 python as well to be consistent with scriptworker-scripts and other repos \o/
Tiny nit: should we ditch python 3.6 along the way or should we keep it for now?

Reply:
I'll ditch 3.6 in the next PR, which I'm working on now. Good call.